core/node/blocksync_request.cpp

```diff
@@ -275,14 +275,15 @@ namespace fc::sync::blocksync {
           depth = kMaxDepth;
         }
 
-        auto binary_request = codec::cbor::encode<Request>(
+        const auto maybe_binary_request = codec::cbor::encode<Request>(
```

Comment:
```suggestion
        auto maybe_binary_request = codec::cbor::encode<Request>(
```
don't `const` what needs to be `move`